### PR TITLE
ACRIOS Systems

### PR DESCRIPTION
**Pull request by mgaspy2**
#### Summary

Info update and device addition before The Things Conference

#### Changes

- Updated device info, images and logo
- Added new device
- Added parser for LMX devices

#### Checklist for Reviewers

- [ ] Title and description should be descriptive (Not just a serial number for example).
- [ ] `profileIDs` should not be `vendorID` and should be a unique value for every profile.
- [ ] All devices should be listed in the vendor's `index.yaml` file.
- [ ] Firmware versions can not be changed.
- [ ] At least 1 image per device and should be transparent.

#### Notes for Reviewers

...

#### Release Notes

- ...

### CODE REVIEW

File: vendor/acrios/acr-cv-101l-m-d.yaml [hunk 1]
```diff
@@ -1,38 +1,90 @@
...
+description: M-Bus to LoRaWAN converter is used to retrofit ANY M-Bus meter with LoRaWAN communication
+
 hardwareVersions:
-  - version: 'v3.7'
+  - version: 'ACR-CV-101L-M-DEAC_v1.1'
     numeric: 1
+
 firmwareVersions:
   - # Firmware version
-    version: 'e8d17c15795f'
```
Comment (LDannijs) [inline vendor/acrios/acr-cv-101l-m-d.yaml:8]: Please do not change the existing values of `hardwareVersions` and `firmwareVersions`.

Comment (mgaspy2) [reply vendor/acrios/acr-cv-101l-m-d.yaml:8]: Fixing it right now

Comment (LDannijs) [reply vendor/acrios/acr-cv-101l-m-d.yaml:8]: Have you ran `make validate fmt` already as well? If not do so please.

Comment (mgaspy2) [reply vendor/acrios/acr-cv-101l-m-d.yaml]: Before commit I ran `make validate`

Comment (LDannijs) [reply vendor/acrios/acr-cv-101l-m-d.yaml]: Could you run make validate fmt too? That will take care of formatting as well.

Comment (mgaspy2) [reply vendor/acrios/acr-cv-101l-m-d.yaml]: Apologies for the oversight earlier. I was multitasking and didn't notice that I missed running `make validate fmt` and the FW/HW versions modifying. I'll take care of it now.

Comment (LDannijs) [reply vendor/acrios/acr-cv-101l-m-d.yaml]: No worries!

Comment (mgaspy2) [reply vendor/acrios/acr-cv-101l-m-d.yaml]: I ran `make validate fwt` and output was:

```
make: *** No rule to make target 'fwt'.  Stop.
```

Comment (mgaspy2) [reply vendor/acrios/acr-cv-101l-m-d.yaml]: Done, git shows no changes

Comment (LDannijs) [reply vendor/acrios/acr-cv-101l-m-d.yaml]: Alright thank you. My colleague @Jaime-Trinidad will do the final check and approval.

**Review by Jaime-Trinidad — APPROVED** [review]

**Pull request merged**